### ISSUE TITLE
Events aren't visible or are thin strips

### ISSUE THREAD

**Issue by osteele** [olinlibrary/abe-web#120]
To reproduce:
* Visit <http://devevents.olin.build/>
* Note that there are events on April 12, 13, 14, 15, 16 etc. (First screenshot.)
* Switch to Day, 3-Day, or Week view
* Note that some the events are missing; some just show as bumps above 9am. (Second screenshot.)

<img width="481" alt="screenshot of safari 4-14-18 7-48-30 am" src="https://user-images.githubusercontent.com/674/38767884-3f615230-3fb8-11e8-9742-a866b03655f2.png">

<img width="1035" alt="screenshot of safari 4-14-18 7-48-06 am" src="https://user-images.githubusercontent.com/674/38767885-41f99912-3fb8-11e8-8005-96580a7a6816.png">

**Comment by kylecombes** [comment]
Huh, that's interesting. Not sure why that is, but I'll take a look.

**Comment by kylecombes** [comment]
Looks like this is only an issue in Chrome/Chromium. Probably CSS related, given that's how positioning works and there is also another Chrome-only CSS bug.

**Comment by kylecombes** [comment]
Should be resolved by https://github.com/kylecombes/ultra-responsive-calendar/commit/2981bd8cd28724460b1b38689a5882dee8ab98d2. Checked in Chromium on Ubuntu.

However, I'm having trouble getting Heroku to update its copy of the `ultra-responsive-calendar` module. It could be in part due to the fact it's being installed from a GitHub repo which doesn't really do versions in the typical npm sense. I'm taking a break from this issue for now. I already tried the suggestions mentioned in [this](https://devcenter.heroku.com/articles/nodejs-support#cache-behavior) and [this](https://coderwall.com/p/jjcpra/clean-heroku-npm-cache).
